recipes/jupyterlab_widgets/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "1.0.0a4" %}
+{% set sha256 = "a51f905de826a15bac5aab995e9f49d1dd4b4b8831f54536b7bad36ee318bbea" %}
+
+package:
+  name: jupyterlab_widgets
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jupyterlab_widgets/jupyterlab_widgets-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging
+
+  run:
+    - python >=3.6
+    - jupyterlab >=3.0.0rc4,<4
+
+test:
+  imports:
+    - jupyterlab_widgets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ipython/ipywidgets
```

We might as well use the URL the `jupyter-widgets` organization?

```suggestion
  home: https://github.com/jupyter-widgets/ipywidgets
```